native/c/zowex.cpp

```diff
@@ -1129,7 +1129,14 @@ int handle_data_set_view_dsn(ZCLIResult result)
 int handle_data_set_list(ZCLIResult result)
 {
   int rc = 0;
-  string dsn = result.get_positional("dsn")->get_value();
+  string dsn = result.get_positional("dsn")->get_value() + ".**";
```

Comment:
What happens if the last qualifier is a single letter in length? The pattern could be 45 characters in length but match a valid dsname that is 44 chars.

Reply:
fixed in 67896c8